### DISCUSSION TITLE
Add Css Styles to DNN Content Library Forms

### DISCUSSION THREAD

**Discussion by Lirhblack** [dnnsoftware/Dnn.Platform#4724] [Q&A]
Hello, anyone know how to access to the HTML or how to add custom styles to DNN content library FORMS? DNN Version 9.04+

**Comment by valadas** [comment]
Are you talkin about the file manager or some other Evoq feature ?

**Reply by Lirhblack** [reply]
Hello, no!, im talking about the new forms in 9.0+ dnn version, 
![forms](https://user-images.githubusercontent.com/63089348/124151594-556b3a80-da58-11eb-89f1-fc2c9fa48d11.png)


Im looking for some kind of access to html or if we have any method to just add a css class to some inputs in the form

Thanks

**Reply by bdukes** [reply]
This is not a DNN Platform feature, it looks like the Form Builder is a feature of Evoq Content and Evoq Engage. You'll need to contact [Evoq Support](https://www.dnnsoftware.com/services/customer-support) for help with this feature.
